### ISSUE TITLE
Do not require users on Windows to set TMPDIR before compile.sh

### ISSUE THREAD

**Issue by thinlizzy** [bazelbuild/bazel#1444]
lberki has pushed a fix for https://github.com/bazelbuild/bazel/issues/1306 but now I am getting a different error, which shows up quicker than all other errors I was having:

```
javac: file not found: \tmp\bazel.WWZok8bR\src\com\google\devtools\build\buildjar\proto\JavaCompilation.java
Usage: javac <options> <source files>
use -help for a list of possible options

Compiling Bazel Java code...

```
but the file seems to be there:
```
John@terrorvision MINGW64 /c/bazel
$ ls /tmp/bazel.WWZok8bR/src/com/google/devtools/build/buildjar/proto/
JavaCompilation.java

```

**Comment by philwo** [comment]
@lberki @dslomov FYI

**Comment by dslomov** [comment]
Per http://www.bazel.io/docs/windows.html, set TMPDIR to smth like `c:/temp`.
That should fix it, but we should also update our scripts so that it is not neccessary

**Comment by thinlizzy** [comment]
I just did a git pull and ..
`Build successful! Binary is here: /c/bazel/output/bazel.exe`

thanks :+1:

**Comment by dslomov** [comment]
Fixed with 19fd76f4376fab4f11e11099aeaf34d36e1d44df.

**Comment by dslomov** [comment]
@thinlizzy Happy to hear that!
